app/shared_features/survey/SurveyModalView.js

```diff
@@ -20,8 +22,17 @@ import {
 
     // TODO: grab the blue color for cancel from a global stylesheet
     _renderNavigation() {
-        if (Platform.OS === 'ios') {
+        if (Platform.OS === 'ios' && !Device.isiPhoneX()) {
             var statusBar = (<View style={{height: 20, width: 9001, backgroundColor: 'black'}}></View>);
+        } else if (Device.isiPhoneX()) {
```

Comment:
- [ ] make it if device is iphonex else if platform == ios to simplify logic so you don't do that extra check